api/fossa/uploadv1.go

```diff
@@ -0,0 +1,86 @@
+package fossa
+
+import (
+	"encoding/json"
+
+	"github.com/apex/log"
+	"github.com/pkg/errors"
+
+	"github.com/fossas/fossa-cli/cmd/fossa/version"
+)
+
+type V1UploadBody struct {
+	VCS struct {
+		Type      string
+		Reference string
+	}
+	Project struct {
+		Title string
+		Id    string
+		Url   string
+		Jira  string
+		Team  string
+	}
+	Revision struct {
+		Id   string
+		Link string
+	}
+	Meta struct {
+		v string
```

Comment:
This should be `V`, otherwise `json.Marshal` will not pick up an unexported field.